src/client/components/dashboard-items/dashboard-items.component.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
-import '../navigation-button/navigation-button.style.css';
+import React, { useContext } from 'react';
+import './dashboard-items.style.css';
 import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { ChartDataContext } from '../../containers/chart-detail-page/chart-detail-context';
 
 const DashboardItems = ({ items }) => {
+  const listItemClicked = useContext(ChartDataContext);
+  const { handleClick } = listItemClicked;
   return (
     <ul className="list">
       {items.map((item) => (
-        <li key={item.id}>{item.value}</li>
+        <li key={item.id}>
+          <Link
+            to="/chartDetail-smartpage"
+            id={item.id}
+            className="a"
+            onClick={handleClick}
+          >
+            {item.value}
+          </Link>
+        </li>
```

Comment:
This is for some reason not reacting properly when clicked (doesn't set `materialId` properly on every click), I'm not sure why, maybe because you're embedding something inside <li>. But it works fine if you replace it with

```
        // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
        <li
          key={item.id}
          id={item.id}
          onClick={(e) => handleClick(e)}
          onKeyDown={handleClick}
        >
          {item.value}
        </li>
```

Reply:
yes we had a problem before but today we solved it in study group we will update it  today